### PR TITLE
Add looping option for player [EDI-905]

### PR DESCRIPTION
**Pull request by IliasHad**

### CODE REVIEW

**Review by brycedev — CHANGES_REQUESTED** [review]
this should be an option at instantiation vs. a method call 

Embed({loop: true})

**Comment by IliasHad** [comment]
@brycedev Can you check it now ?

File: src/Embed.ts [hunk 1]
```diff
@@ -208,6 +220,7 @@ class Embed {
     setTimeout(() => {
       this.editframeLogo.remove()
     }, 500)
+    this.handlePlayerLoop()
```
Comment (brycedev) [inline src/Embed.ts:223]: we don't want to call this during handleEditorReady

looping should just work at instantiation by passing it into preview, not triggering it from a method

**Review by brycedev — CHANGES_REQUESTED** [review]

File: src/Embed.ts [hunk 2]
```diff
@@ -200,6 +203,15 @@ class Embed {
       })
     }
   }
+  private handlePlayerLoop = async () => {
+    if (this.loop) {
+      console.log(await this.sendCallWithValue("setPlayerLoop", {
```
Comment (brycedev) [inline src/Embed.ts:208]: remove this console.log

Comment (IliasHad) [reply src/Embed.ts:208]: Okey

File: src/Embed.ts (hunk 1 above)
Comment (IliasHad) [reply src/Embed.ts:223]: I tried to have it as config but it doesn't seem to work on my end, maybe the iframe wasn´t ready to send the event to it

**Pull request closed without merging**